nomad/csi_endpoint.go

```diff
@@ -189,7 +189,7 @@ func (v *CSIVolume) Get(args *structs.CSIVolumeGetRequest, reply *structs.CSIVol
 				return err
 			}
 			if vol != nil {
-				vol, err = state.CSIVolumeDenormalize(ws, vol)
+				vol, err = state.CSIVolumeDenormalize(ws, vol.Copy())
```

Comment:
fwiw, we only do this to volumes that we have previously `CSIVolumeByID` which is already a copy, so we don't need the second copy. That is error prone, though.